sktime/utils/_testing/forecasting.py

```
@@ -29,6 +29,8 @@ def _get_n_columns(tag):
         n_columns_list = [2]
     elif tag == "both":
         n_columns_list = [1, 2]
+    else:
+        n_columns_list = [1]
```

I think else it should raise an error, since this is an unexpected input!